### PR TITLE
Update zhCN

### PR DESCRIPTION
**Pull request by Mini-Dragon**
## Issue references

Fixes #

## Proposed changes

Update zhCN

### CODE REVIEW

File: Localization/Translations/MinimapIcon/MinimapIcon.lua [hunk 1]
```diff
@@ -8,7 +8,7 @@ local minimapIconLocales = {
         ["koKR"] = "메뉴 표시",
         ["esMX"] = "Mostrar menú",
         ["zhCN"] = "切换菜单",
```
Comment (BreakBB) [inline Localization/Translations/MinimapIcon/MinimapIcon.lua:10]:
```suggestion
        ["zhCN"] = "打开主菜单",
```

File: Localization/Translations/MinimapIcon/MinimapIcon.lua [hunk 2]
```diff
@@ -8,7 +8,7 @@ local minimapIconLocales = {
...
-        ["enUS"] = true,
+        ["enUS"] = "打开主菜单",
```
Comment (BreakBB) [inline Localization/Translations/MinimapIcon/MinimapIcon.lua:11]:
```suggestion
        ["enUS"] = true,
```

**Comment by BreakBB** [comment]
Thank you very much @Mini-Dragon for taking the time to provide these translations, we appreciate it! 🙏🏻

File: Localization/Translations/Options/DBMHud.lua [hunk 3]
```diff
@@ -81,7 +81,7 @@ local dbmHudOptionsLocales = {
         ["koKR"] = false,
         ["esMX"] = "Mostrar iconos de interacción de objetos",
         ["enUS"] = true,
-        ["zhCN"] = false,
+        ["zhCN"] = "显示目标交互图标"
```
Comment (BreakBB) [inline Localization/Translations/Options/DBMHud.lua:84]:
```suggestion
        ["zhCN"] = "显示目标交互图标",
```

File: Localization/Translations/Options/Icons.lua [hunk 4]
```diff
@@ -873,7 +873,7 @@ local iconOptionsLocales = {
         ["koKR"] = false,
         ["esMX"] = "Mostrar iconos para...",
         ["enUS"] = true,
-        ["zhCN"] = false,
+        ["zhCN"] = "显示...图标""",
```
Comment (BreakBB) [inline Localization/Translations/Options/Icons.lua:876]:
```suggestion
        ["zhCN"] = "显示...图标",
```

**Comment by Mini-Dragon** [comment]
There is a typo, please fix it.

**Comment by BreakBB** [comment]
> There is a typo, please fix it.

Where exactly?

**Comment by Mini-Dragon** [comment]
It's good now. You fixed that comma thing.

**Pull request merged**